exposed/src/jvmMain/kotlin/kotlinx/uuid/exposed/KotlinxUUIDEntity.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2020-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.uuid.exposed
+
+import kotlinx.uuid.*
+import org.jetbrains.exposed.dao.*
+import org.jetbrains.exposed.dao.id.EntityID
```

Comment:
No star import, because Exposed has moved `EntityID` from `dao` to `dao.id`. For binary compatibility, the first definition is kept.